tests/plugins/test_koji_upload.py

```diff
@@ -938,200 +967,14 @@ def test_koji_promote_image_tags(self, tmpdir, os_env, additional_tags):
 
         assert set(tags) == expected_tags
 
-    @pytest.mark.parametrize(('apis',
-                              'docker_registry',
-                              'pulp_registries',
-                              'metadata_only',
-                              'blocksize',
-                              'target'), [
-        ('v1-only',
-         False,
-         1,
-         False,
-         None,
-         'images-docker-candidate'),
-
-        ('v1+v2',
-         True,
-         2,
-         False,
-         10485760,
-         None),
-
-        ('v2-only',
-         True,
-         1,
-         True,
-         None,
-         None),
-
-        ('v1+v2',
-         True,
-         0,
-         False,
-         10485760,
-         None),
-
-    ])
-    @pytest.mark.parametrize(('has_config', 'is_autorebuild'), [
-        (True,
-         True),
-        (False,
-         False),
-    ])
-    @pytest.mark.parametrize('tag_later', (True, False))
-    def test_koji_promote_success(self, tmpdir, apis, docker_registry,
-                                  pulp_registries, metadata_only, blocksize,
-                                  target, os_env, has_config, is_autorebuild,
-                                  tag_later):
```

Comment:
We still want this test. It should just do a bit less. It should:

- set up the mock environment and run the plugin of course
- verify the 'metadata_version', 'buildroots', and 'output' keys as before
- not verify 'build', as this is something that koji-import will create